public-ui/src/app/views/survey-view/survey-view.component.css

```diff
@@ -4,9 +4,6 @@ strong {
 }
 .title-container {
     width: 100%;
-    display: flex;
```

Comment:
This is to address the survey helptext being displayed on the same line as the survey title.

Reply:
add this under .title-container{...} : 
.title-container p {
    width: 100%;
}